runtime/src/bank.rs

```diff
@@ -8486,8 +8486,9 @@ pub mod test_utils {
         super::Bank,
         solana_sdk::{hash::hashv, pubkey::Pubkey},
         solana_vote_program::vote_state::{self, BlockTimestamp, VoteStateVersions},
+        std::sync::Arc,
     };
-    pub fn goto_end_of_slot(bank: &Bank) {
+    pub fn goto_end_of_slot(bank: Arc<Bank>) {
```

Comment:
yes, taking ownership of `Arc<Bank>` is intentional for upcoming changes. strictly speaking, implied cloning isn't neeed, however i leaned for coding simplicity over perfection.

after all, this isn't part of hot path, rather a tiny and aged test helper function... who cares for additional clones for it? ;)